config/environments/production.rb

```diff
@@ -82,6 +82,8 @@
     authentication: :plain,
   }
 
+  config.x.WillPaginate.per_page = 15
```

Comment:
Like the x 👍
Any reason you departed from snake case for the WillPaginate? (i.e why not `will_paginate`?)

Or even just `config.x.results_per_page`? The variable name should be whatever is meaningful to your app and your code, and it shouldn't assume the usage. What if you later decide you need your own code (i.e. not just will_paginate gem) to access this config value for some reason or what if you chose to change the pagination gem you're using? Make the name descriptive, but not specific to particular code implementations.

Reply:
I followed the readme for will paginate...
```
# set per_page globally
WillPaginate.per_page = 10
```
I didn't realise that using config.x.variable_name was possible and just setting a variable. 

Will make these changes :)